README.md

```diff
@@ -84,7 +84,7 @@ async function main() {
       console.log(err.name); // BadRequestError
       console.log(err.headers); // {server: 'nginx', ...}
     } else {
-      throw err;
+      return Promise.reject(err);
     }
   });
```

Comment:
I think the better thing to do here would be:

```ts
async function main() {
  const liveStream = await mux.video.liveStreams.create().catch(async (err) => {
    if (err instanceof Mux.APIError) {
      console.log(err.status); // 400
      console.log(err.name); // BadRequestError
      console.log(err.headers); // {server: 'nginx', ...}
    } else {
      throw err;
    }
  });
```

cc @yjp20

Reply:
I don't think mixing two styles of code is a good idea. I suggest only use async style here.